joss_paper/pyfixest_joss.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'PyFixest: A Python Port of fixest for High-Dimensional Fixed-Effects Regression'
+date: 8 August 2025
+author:
+  - name: Alexander Fischer
+    affiliation: Trivago
+---
+
+
+## Summary
```

Comment:
I know it's hard, but we should somehow prove the already important adoption of the package. Either by the number of downloads and / or some testimonies of companies that use it (I can testify we used it for price experimentation at Wolt)